script/mapping-tables.js

```diff
@@ -229,28 +229,7 @@ function mappingTables() {
 
     var expandCollapseDetails = function (detCont, action) {
       queryAll('details', detCont).forEach(function (details) {
-        var detailsSummary = details.querySelector('summary');
-        var detailsNotSummary = Array.prototype.slice
-          .call(details.children)
-          .filter(function (child) {
-            return child !== detailsSummary;
-          });
-
-        if (action == 'collapse') {
-          details.classList.remove('open');
-          details.open = false;
-          detailsSummary.setAttribute('aria-expanded', false);
-          detailsNotSummary.forEach(function (element) {
-            hideElement(element);
-          });
-        } else {
-          details.classList.add('open');
-          details.open = true;
-          detailsSummary.setAttribute('aria-expanded', true);
-          detailsNotSummary.forEach(function (element) {
-            showElement(element);
-          });
-        }
+        details.open = action !== 'collapse'
```

Comment:
@jnurthen do we still need this polyfill stuff? [According to MDN, IE doesn't support details summary](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#browser_compatibility)... is that ok? If not, I will back this part of the change out and fix the issue differently